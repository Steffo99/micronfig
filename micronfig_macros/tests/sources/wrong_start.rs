micronfig::config! {
	GARASAUTO: i64,
}

fn main() {
	std::env::set_var("GARASAUTO", "-1");
	println!("{:#?}", GARASAUTO());
}
