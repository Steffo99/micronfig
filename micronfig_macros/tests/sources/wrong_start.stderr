error: first type of a conversion chain should always be `String` or `std::string::String`, type aliases are not allowed
 --> tests/sources/wrong_start.rs:2:13
  |
2 |     GARASAUTO: i64,
  |                ^^^

error[E0425]: cannot find function, tuple struct or tuple variant `GARASAUTO` in this scope
 --> tests/sources/wrong_start.rs:7:19
  |
7 |     println!("{:?}", GARASAUTO())
  |                      ^^^^^^^^^ not found in this scope
